pom.xml

```diff
@@ -2,12 +2,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.camunda.community</groupId>
+    <artifactId>community-hub-release-parent</artifactId>
+    <version>1.2.2</version>
+  </parent>
+
   <groupId>io.camunda</groupId>
-  <artifactId>camunda-cloud-testing</artifactId>
+  <artifactId>zeebe-bpmn-assert</artifactId>
```

Comment:
Do we want to reflect the name change also in the package structure?

Reply:
Yes, we will do this in a different PR as discussed